### PR TITLE
feat: fellowship basic implementation

### LINKED ISSUE #2307: Fellowship. Add structures, implement Members part
- Add common domain functions, constants, and types;
- Implement the Members model by connecting it to the new transport layer;
- Implement basic UI for Members' feature

### PR DESCRIPTION
**Pull request by sokolova-an**
Added navigation for Fellowship;
Added search with filters and profile to fellowship page
Added referendum list
Added members modal

### CODE REVIEW

File: src/renderer/pages/Fellowship/ui/Fellowship.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { useGate } from 'effector-react';
+
+import { useI18n } from '@app/providers';
+import { Header } from '@shared/ui';
+import { Members } from '@/features/fellowship/memebers/ui/Members';
```
Comment (johnthecat) [inline src/renderer/pages/Fellowship/ui/Fellowship.tsx]: typo

File: src/renderer/pages/Fellowship/aggregates/fellowshipPage.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+import { type ChainId } from '@/shared/core';
+import { type CollectivePalletsType } from '@/domains/collectives/lib/types';
+import { fellowshipNetworkAggregate } from '@/features/fellowship/network/aggregates/fellowshipNetwork';
+
+const COLLECTIVES_CHAIN_ID: ChainId = '0x46ee89aa2eedd13e988962630ec9fb7565964cf5023bb351f2b6b25c1b68b0b2';
+
+const flow = createGate<{ chainId: ChainId }>();
+
+// We want to show main fellowship on specific chain - Polckadot Collectives
```
Comment (johnthecat) [inline src/renderer/pages/Fellowship/aggregates/fellowshipPage.ts]: typo

**Review by johnthecat — APPROVED** [review]

File: src/renderer/domains/collectives/model/members/model.ts [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+      currentAbortController = new AbortController();
+
+      const collectiveMembers = await collectivePallet.storage.members(palletType, api);
+      if (currentAbortController.signal.aborted) return;
+
+      const coreMembers =
+        palletType === 'fellowship'
+          ? await fellowshipCorePallet.storage.member(api)
+          : await ambassadorCorePallet.storage.member(api);
+      if (currentAbortController.signal.aborted) return;
```
Comment (sokolova-an) [inline src/renderer/domains/collectives/model/members/model.ts]: As discussed,  let's change `ambassadorCorePallet`  to `collectivesCorePallet`, so it won't be confusing later with other collectives

File: src/renderer/domains/collectives/model/referendum/mapper.ts [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import { type ReferendaReferendumInfoConvictionVotingTally, type ReferendumId } from '@/shared/pallet/referenda';
+
+import { type Referendum } from './types';
+
+export const mapReferendum = (id: ReferendumId, info: ReferendaReferendumInfoConvictionVotingTally): Referendum => {
+  switch (info.type) {
+    case 'Ongoing':
+      if (!('bareAyes' in info.data.tally)) {
+        throw new Error(
+          `Fellowship of Ambassador tally is incorrect, got\n${JSON.stringify(info.data.tally, null, 2)}`,
```
Comment (sokolova-an) [inline src/renderer/domains/collectives/model/referendum/mapper.ts]: maybe just: 'Collective tally is incorrect...'

File: src/renderer/features/fellowship-members/constants.ts [hunk 5]
```diff
@@ -0,0 +1,3 @@
+export const error = {
```
Comment (sokolova-an) [inline src/renderer/features/fellowship-members/constants.ts]: shouldn't it be from the capital letter?

File: src/renderer/features/fellowship-referendum-details/model/status.ts [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+//   filter: fellowshipNetworkFeature.model.network.$isActive,
+//   target: referendumsFeatureStatus.restore,
+// });
+//
+// sample({
+//   clock: [fellowshipNetworkFeature.model.network.$isDisconnected, referendumsFeatureStatus.start],
+//   filter: fellowshipNetworkFeature.model.network.$isDisconnected,
+//   fn: () => ({ type: 'warning' as const, error: new Error(error.networkDisabled) }),
+//   target: referendumsFeatureStatus.fail,
+// });
```
Comment (sokolova-an) [inline src/renderer/features/fellowship-referendum-details/model/status.ts]: do we need this code?

File: src/renderer/features/fellowship-referendums/components/List/index.ts [hunk 7]
```diff
@@ -0,0 +1,2 @@
+// export { OngoingReferendums } from './OngoingReferendums';
```
Comment (sokolova-an) [inline src/renderer/features/fellowship-referendums/components/List/index.ts]: can remove it for now

**Review by sokolova-an — COMMENTED** [review]
looks good

**Pull request merged**